docs/reference/api-endpoint-reference.md

```
@@ -94,6 +94,7 @@ curl -X POST http://localhost:8000/scan-upload \
 | `enable_meta` | `bool` |
 | `llm_consensus_runs` | `int` |
 | `llm_max_tokens` | `int | None` |
+| `llm_reasoning_effort` | `LLMReasoningEffort | None` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)